### DISCUSSION TITLE
One Drive Business - Can't get a shared folder synced

### DISCUSSION THREAD

**Discussion by Peter-Ries** [abraunegg/onedrive#1889] [Q&A] [answered]
Hi, I have a One Drive Business account via my company.

I added a Sharepoint directory with some documentation via right click in browser to my One Drive where I can see it.

But I can't get this folder into sync process. All regular folders are being synced.

As of documentation I should use "onedrive --list-shared-folders" but this returns no items.

Where can I configure this?

config in my home is default - just changed location to another subdir

sync_list file contains 
-Archive/
-Microsoft*/
-notebooks/
Documentation/

where the last entry is the one I need. Tried with / without / with leading slash / with trainling asterix...

No clue left. Can anybody help? Thankx!

**Comment by abraunegg** [comment, marked as answer]
@Peter-Ries 
> I added a Sharepoint directory with some documentation via right click in browser to my One Drive where I can see it.

Adding a SharePoint folder using the "Add to my OneDrive" is not supported by the OneDrive API. You cannot access the folder in this way - this is a Microsoft limitation. See: https://github.com/OneDrive/onedrive-api-docs/issues/1427

You can only access SharePoint folders by following https://github.com/abraunegg/onedrive/blob/master/docs/SharePoint-Shared-Libraries.md

> As of documentation I should use "onedrive --list-shared-folders" but this returns no items

This is for OneDrive Business Shared Folders - this is *not* for SharePoint Sites
